src/PaddleEnum.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Paddle\SDK;
+
+use MyCLabs\Enum\Enum;
+
+class PaddleEnum extends Enum
+{
+    protected const Undefined = null;
+
+    public static function from($value): static
+    {
+        try {
+            return parent::from($value);
+        } catch (\UnexpectedValueException $e) {
+            if ($value === null) {
+                throw $e;
```

Comment:
I think we should we add a `@throws` doc block so it's clear in IDEs 👍

Reply:
✅ done